Makefile

```diff
@@ -1,9 +1,5 @@
-all:
-	go install -v ./...
-	go test -c -v ./inventory
-	go test -c -v ./server
```

Comment:
CI fails due to that target being removed.
It should be something like:
```
all:
	go install ./...
	go test -i ./...
```

Reply:
It fails because now CI runs tests before it didn't. Previously it just built test binaries.

Reply:
Yeah, that's what I wrote?.. I mean, I know that. Now can we actually fix CI instead of explaining again why it fails?
It is a good idea to at least check that code can be compiled. We can also build test binaries just to check that they can be compiled too. There is no reason to actually run tests in Travis CI – that's what @puneet0191 is doing on Jenkins.

Reply:
yes please, just test if the binaries are being built, you will still need an instance against which you need to run those tests, do you have that setup with Travis CI ?

Reply:
> you will still need an instance against which you need to run those tests, do you have that setup with Travis CI ?

No. The idea is to do that only on Jenkins.